toolkit/tools/scheduler/scheduler.go

```diff
@@ -63,6 +63,7 @@ var (
 	noCleanup            = app.Flag("no-cleanup", "Whether or not to delete the chroot folder after the build is done").Bool()
 	noCache              = app.Flag("no-cache", "Disables using prebuilt cached packages.").Bool()
 	stopOnFailure        = app.Flag("stop-on-failure", "Stop on failed build").Bool()
+	hydratedBuild        = app.Flag("hydrated-build", "Build individual packages with Hydrated RPMs").String()
```

Comment:
This should be a Bool() rather than a String(). (See the other PR for an example)